i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/messages.mdx

```diff
@@ -371,16 +371,16 @@ Mit Hilfe der Formatierung können Sie Ihre Nachrichten in Famedly detaillierter
 
 Wenn Sie Famedly auf Ihrem Desktop oder mobil verwenden, können Sie Nachrichten mit diesen Befehlen formatieren.
 
-| Format                     | Beschreibung                                                                                                  |
-| -------------------------- | ------------------------------------------------------------------------------------------------------------- |
-| **Fett**                   | Text mit Sternen umgeben: `**Ihr Text**`                                                                      |
-| _Kursiv_                   | Text mit Unterstrichen umgeben: `_Ihr Text_`                                                                  |
-| ~Durchgestrichen~          | Text mit Tilden umgeben: `~Ihr Text~`                                                                         |
-| `Code`                     | Text mit Backticks umgeben: `` `Ihr Text` ``                                                                  |
-| Zitat                      | Add an angled bracket in front of text: `>your text`                                                          |
-| Code block                 | Fügen Sie drei Backticks vor und nach dem Text hinzu: `Ihr Text`                                              |
-| Überschrift 1              | Add `#` then press Space in front of text                                                                     |
-| Überschrift 2              | Add `##` then press Space in front of text                                                                    |
-| Überschrift 3              | Add `###` then press Space in front of text                                                                   |
-| 1. Nummerierte Liste       | Schreiben Sie 1 und einen Punkt und dann ein Leerzeichen vor dem Text: 1. Ihr Text.                           |
-| - Aufzählungsliste         | Fügen Sie ein Sternchen und ein Leerzeichen vor Ihrem Text ein.                                               |
+| Format                     | Beschreibung                                                                          |
+| -------------------------- | --------------------------------------------------------------------------------------|
+| **Fett**                   | Text mit Sternen umgeben: `**Ihr Text**`                                              |
+| _Kursiv_                   | Text mit Unterstrichen umgeben: `_Ihr Text_`                                          |
+| ~Durchgestrichen~          | Text mit Tilden umgeben: `~Ihr Text~`                                                 |
+| `Code`                     | Text mit Backticks umgeben: `` `Ihr Text` ``                                          |
+| Zitat                      | Setzen Sie eine spitze Klammer vor den Text: `>Ihr Text`                              |
```

Comment:
Vor dem - oder?

Reply:
Nein, "vor den Text" ist in dem Kontext richtig

Reply:
Du setzt etwas "vor den Text", aber etwas steht "vor dem Text". Glaub mir, das ist richtig so